src/StackExchange.Redis/RedisFeatures.cs

```diff
@@ -253,6 +253,11 @@ public RedisFeatures(Version version)
         /// </summary>
         public bool PushMultiple => Version >= v4_0_0;
 
+        /// <summary>
+        /// Is LMOVE available?
+        /// </summary>
+        public bool ListMove => Version >= v6_2_0;
```

Comment:
Let's remove this and move to the IsBelow skip check version in main, can see examples now and in a neighboring PR to make it global!